lib/Test2/API.pm

```diff
@@ -596,9 +596,9 @@ generated by the test system:
         my_ok(0, "fail");
     };
 
-    my_ok(@$events == 2, "got 2 events, the pass and the fail");
```

Comment:
notice that ok() is never imported in this example. The example is intended to demonstrate that intercept { ... } lets you use your own test tools to test themselves.